src/pages/api/graphql.tsx

```diff
@@ -107,27 +98,28 @@ export const apolloServer = new ApolloServer({
           // TODO log error
           // console.error('error in creating new user', e)
 
-          throw new ApolloError('Error creating new user', e as string)
+          throw new GraphQLError('Error in creating user', {
+            extensions: { code: 'INTERNAL_SERVER_ERROR' },
+          })
         }
       }
 
-      // Set db connection, keystone api, and user in context
-
-      return { db, user: loggedInUser }
+      // Set db connection, keystone api, data sources, and user in context
+      return {
```

Comment:
In addition to the db and user, we're now passing our dataSources in here. They'll be accessible in the resolvers just as before.